engine/utils/phase_debug_webapp/src/main.rs

```diff
@@ -0,0 +1,5 @@
+use phase_debug_webapp::*;
+
+fn main() {
+    run(|| std::fs::read_to_string("/tmp/hax_debug/debug-hax-engine.json").unwrap())
```

Comment:
A hard-coded path? I'm not sure that's a good idea....

Reply:
Oops, that's was not intended

Reply:
Fixed it: I actually removed this binary entirely, it has no purpose now